executor/src/native_executor.rs

```diff
@@ -160,12 +162,12 @@ mod tests {
 
 		let transactions = txs.into_iter().map(|transaction| {
 			let signature = secret_for(&transaction.signed).unwrap()
-				.sign(&transaction.to_vec())
-				.inner();
+				.sign(&transaction.to_vec());
+
 			UncheckedTransaction { transaction, signature }
 		}).collect::<Vec<_>>();
 
-		let transaction_root = ordered_trie_root(transactions.iter().map(Slicable::to_vec)).0;
+		let transaction_root = H256(ordered_trie_root(transactions.iter().map(Slicable::to_vec)).0);
```

Comment:
hmm.. the result of `ordered_trie_root` is an `H256`, just from another crate :(

Reply:
yep, we could use the `ethereum-types` crate but I don't think Parity can use the no-std variant of it.